FreeChat/FreeChat/Views/Pages/ConversationsPage.xaml

```diff
@@ -71,7 +71,7 @@
                             <ColumnDefinition Width="Auto" />
                         </Grid.ColumnDefinitions>
 
-                        <Entry Placeholder="Search"
```

Comment:
Please, don't do such changes, if you're willing to experiment with the app, do that on your fork.